kodi_game_scripting/process_game_addons.py

```diff
@@ -344,24 +355,57 @@ def process_addon_files(self):
         """ Generate addon files """
         self.load_assets()
         self.load_library_file()
+
+        gitrepo = GitRepo(GitHubRepo(self.game_name, '', ''),
+                          os.path.join(self._args.working_directory, 'build',
+                                       'build', self.game_name, 'src'))
+        if gitrepo.is_git_repo():
+            self.info['repo_hexsha'] = gitrepo.get_hexsha()
+
         template_processor.TemplateProcessor.process(
             'addon', self.path, self.info)
 
+    def bump_version(self):
+        """ Bump version with pkg version from Git """
+        if self._repo:
+            git_tag = self._repo.describe()
+            match = re.search(r'^(?:[0-9]+\.){3}([0-9]+)', git_tag)
+            pkg_version = match.group(1) if match else '100'
```

Comment:
Why start with 100?
Also this method doesn't bump anything, like the function name advertises.

Reply:
I've seen that libreelec does it in a similar way (https://github.com/LibreELEC/LibreELEC.tv/blob/master/packages/mediacenter/kodi-binary-addons/game.libretro.snes9x/package.mk#L7). But we can just start at 0 if you prefer.

Reply:
I'll rework the implementation once I know what we want to have. This has been just hacked together yesterday evening.